### PR TITLE
Create red_river_parish.json

### PR DESCRIPTION
**Pull request by marcale**
Red River Parish datasource (addresses).

### CODE REVIEW

File: sources/us/la/red_river_parish.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "coverage": {
+        "US Census": {"geoid": "22081", "name": "Red River Parish", "state": "Louisiana"},
+        "country": "us",
+        "state": "la",
+        "county": "Red River"
+    },
+    "data": "http://www.redriver911.com/wp-content/uploads/2016/03/911_ADDRESS_Public.zip",
+    "website": "http://www.redriver911.com/data/",
+    "type": "http",
+    "accuracy":"1",
```
Comment (iandees) [inline sources/us/la/red_river_parish.json]: `accuracy` should be inside the `conform` object.

File: sources/us/la/red_river_parish.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "compression": "zip",
+    "conform": {
+        "type": "shapefile",
+        "number": "NUMERICAL",
+        "street": "Street",
+        "city": "Add_City",
+        "postcode": "Add_Zip",
+        "lon":"Logitude",
```
Comment (iandees) [inline sources/us/la/red_river_parish.json]: We automatically extract the latitude and longitude from shapefiles, so the `lat` and `lon` keys should be removed in this case.

**Comment by slibby** [comment]
thanks @marcale, this looks great -- can you add "region": "Address_st" to your conform so it will get the state name in there? Thanks!

**Comment by marcale** [comment]
@slibby @iandees Is everything fine with this version? Can it be merged with the master branch? Should I ask somebody to merge it or there is an option for me to do it?

**Comment by iandees** [comment]
I can do it! Thanks for your contribution.

**Comment by iandees** [comment]
(And thanks for following up with us)

**Pull request merged**